src/control-plane/backend/lambda/api/store/aws/flink.ts

```diff
@@ -47,34 +48,35 @@ export const updateFlinkApplication = async (
   streamingSinkKinesisConfig: any[],
 ) => {
   try {
-    if (
-      (!streamEnableAppIds || streamEnableAppIds.length === 0) &&
-      application.ApplicationDetail?.ApplicationStatus ===
-        ApplicationStatus.RUNNING
-    ) {
-      // Disable all the streams, stop flink application
-      logger.debug(
-        'Disabling all the streams and stopping the Flink application',
+    if (isEmpty(streamEnableAppIds)) {
```

Comment:
We should cover those changes with tests.

Reply:
done